### DISCUSSION TITLE
Select with creatable: will this be available in Paste?

### DISCUSSION THREAD

**Discussion by elainelylee** [twilio-labs/paste#615] [General] [answered]
Console react has a component where user can create or select an item via the "Select with creatable" component. Will this be in Paste? Didn't see it explicitly in the roadmap. If not, how should we create this component? 
![image](https://user-images.githubusercontent.com/6718764/88762577-526c1980-d126-11ea-93cd-f15f43754252.png)

**Comment by kate-porter** [comment]
Hey @elainelylee - you might be able to accomplish something similar to this with combobox, but I'll need to get the rest of the team involved in this convo because I'm not certain. We'll report back soon!

**Comment by kate-porter** [comment, marked as answer]
Hey @elainelylee - an update to our combobox component will be merged in today or tomorrow that will allow you to accomplish this. The options in a combobox are entirely customizable. We don't have the symbols added to Sketch quite yet (it will likely be added in our next sprint, starting today, or the following sprint), but in the meantime, you can just use the select symbol and the menu symbol to compose this in your designs.

When you get to the build stage for this, this is how your engineers can do it: add a “create” option at the end of the items array, use the option template to render the “create” based on the item, and—using the state of the input value—you can append what you’ve typed to the "create" option.
